### DISCUSSION TITLE
[font] fontSize global 설정

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#137] [Ideas]
현재 태그에 대해 각기 다른 폰트사이즈를 지정하고 있는데 이를 하나로 통일하였으면 좋겠습니다.
예를 들면 h2 : fontSize: 24px .. 이런 식으로 하고 피그마에 현재 폰트가 다른 부분은 그냥 어떤 태그를 쓸 건지만 적어주면 덜 번거로울 것 같습니다 (남경님 항상 디자인 감사합니당 🥲🥲🥰❤️)

**Comment by YuHyun-P** [comment]
폰트 통일 좋습니다!! 
만약 통일한다면, 남경님의 힘듦을 덜고자 각자 페이지에서 사용되는 폰트들을 확인해서 통일하는 작업을 할까요👀?
